engine/api/src/main/scala/pl/touk/nussknacker/engine/api/context/ProcessCompilationError.scala

```diff
@@ -141,6 +141,14 @@ object ProcessCompilationError {
       EmptyMandatoryParameter(paramName, nodeId.id)
   }
 
+  case class BlankParameter(paramName: String, nodeId: String)
```

Comment:
I think we shouldn't have to change this class for each custom validator...  They should define their errors where they are defined...

Reply:
Changed to `ErrorValidationParameter(validator, paramName, nodeId)`

Reply:
All errors are defined at: `PrettyValidationErrors`. I've added `case class ErrorValidationParameter(validator: ParameterValidator, paramName: String, nodeId: String)`. No we create message and descripton checking type validator class.